utils/eventtracking.js

```diff
@@ -90,11 +109,23 @@ const askForConsentIfNeeded = async (options) => {
     const shellSettings = settings.getShellSettings();
     // if the appropriate option is not in settings, ask now and save settings.
     if (!(TRACKING_ENABLED_KEY in shellSettings)) {
-        shellSettings[TRACKING_ENABLED_KEY] = shouldOptInByDefault() || await getEventTrackingConsent();
-        shellSettings[TRACKING_SESSION_ID_KEY] = shellSettings[TRACKING_ENABLED_KEY] ? uuid.v4() : undefined;
+        shellSettings[TRACKING_ENABLED_KEY] =
```

Comment:
is this intentional change of formatting?

Reply:
I think so, this is all after I command `yarn fix`